database.yml

```diff
@@ -3,7 +3,7 @@ development:
   database: fixme_backend_development
   user: postgres
   password: makis123
-  url: {{ envOr "DATABASE_URL" "postgres://postgres:postgres@127.0.0.1:5432/fixme_backend_development?sslmode=disable"}}
```

Comment:
Could you revert this change because it's going to break the production env?

Reply:
my bad i missed that case